.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please add linter for HTML, CSS, and Javascript from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config.